austation/code/__DEFINES/diseases.dm

```diff
@@ -0,0 +1,2 @@
+// Custom spread flag that prevents disease being caught by catgirls/lizards when *ingested*
+#define DISEASE_SPREAD_STRONG_STOMACH	(1<<7)
```

Comment:
Name is a bit misleading, I had to read the flag toggle a few times to understand, but maybe that's just me. 

Maybe specifying immunity in the name might help a bit? i.e `DISEASE_CHECK_STRONG_STOMACH`